overlord/snapshotstate/snapshotstate.go

```diff
@@ -330,6 +343,16 @@ func Import(ctx context.Context, st *state.State, r io.Reader) (setID uint64, sn
 		if dupErr, ok := err.(backend.DuplicatedSnapshotImportError); ok {
 			st.Lock()
 			defer st.Unlock()
+			// XXX: ideally, we should retry import.
```

Comment:
yes, I think the easier to explain API behavior is that if we hit a conflict we simplify retry the import with a flag that doesn't detect duplicate snapshots anymore